CONTRIBUTING.md

```diff
@@ -1,35 +1,35 @@
 # Contributing
 
 <!-- TOC -->
-* [Contributing](#contributing)
-  * [Submitting a Change](#submitting-a-change)
-    * [Commit Message Convention](#commit-message-convention)
-    * [Common Commit Types](#common-commit-types)
-    * [Breaking Changes](#breaking-changes)
-      * [What is a breaking change](#what-is-a-breaking-change)
-      * [How to approach breaking changes](#how-to-approach-breaking-changes)
-      * [Marking changes as breaking](#marking-changes-as-breaking)
-      * [Removing deprecated functionality](#removing-deprecated-functionality)
-    * [Reverting Commits](#reverting-commits)
-    * [Highlight your code snippets in your messages](#highlight-your-code-snippets-in-your-messages)
-  * [Release Process](#release-process)
-    * [1) Feature Branches (regular changes)](#1--feature-branches--regular-changes-)
-    * [2) Pre-Releases (Beta / Release Candidate branches for complex changes)](#2--pre-releases--beta--release-candidate-branches-for-complex-changes-)
-    * [3) Experimental Releases](#3--experimental-releases)
-<!-- TOC -->
+
+- [Contributing](#contributing)
+  - [Submitting a Change](#submitting-a-change)
+    - [Commit Message Convention](#commit-message-convention)
+    - [Common Commit Types](#common-commit-types)
+    - [Breaking Changes](#breaking-changes)
+      - [What is a breaking change](#what-is-a-breaking-change)
+      - [How to approach breaking changes](#how-to-approach-breaking-changes)
+      - [Marking changes as breaking](#marking-changes-as-breaking)
+      - [Removing deprecated functionality](#removing-deprecated-functionality)
+    - [Reverting Commits](#reverting-commits)
+    - [Highlight your code snippets in your messages](#highlight-your-code-snippets-in-your-messages)
+  - [Release Process](#release-process)
+  _ [1) Feature Branches (regular changes)](#1--feature-branches--regular-changes-)
+  _ [2) Pre-Releases (Beta / Release Candidate branches for complex changes)](#2--pre-releases--beta--release-candidate-branches-for-complex-changes-) \* [3) Experimental Releases](#3--experimental-releases)
```

Comment:
🟢 Why are these 3 underscores?

Reply:
Prettier did this. Seems like it got it wrong 🤷. I've fixed it now.